src/dataviews/dataview-model-base.js

```diff
@@ -167,8 +159,14 @@ module.exports = Model.extend({
   },
 
   fetch: function (opts) {
+    opts = opts || {};
     this.trigger('loading', this);
-    return Model.prototype.fetch.call(this, opts);
+    return Model.prototype.fetch.call(this, {
```

Comment:
There are more attributes accepted in the fetch options (`complete` for example). Aren't we missing that?